sklearn/model_selection/_validation.py

```diff
@@ -106,19 +114,49 @@ def cross_val_score(estimator, X, y=None, groups=None, scoring=None, cv=None,
 
     Returns
     -------
-    scores : array of float, shape=(len(list(cv)),)
+    scores : array of float or dict of such arrays, array shape=(n_splits,)
```

Comment:
Well but this PR half implements it. We return a dict for the multiple metrics case. If we do that, why not go all the way?